gym_ignition/base/robot/robot_links.py

```diff
@@ -84,3 +84,23 @@ def link_acceleration(self, link_name: str) -> Tuple[np.ndarray, np.ndarray]:
             - linear acceleration: a 3D array in the [ax, ay, az] form.
             - angular acceleration: a 3D array in the [wdotx, wdoty, wdotz] form.
         """
+
+    def apply_external_force(self,
+                             link_name: str,
+                             force: np.ndarray,
+                             torque: np.ndarray) -> bool:
+        """
+        Apply external force and torque to the link.
+
+        The force and the torque are applied at the link origin.
+        The force is expressed in the world frame.
+        The torque is expressed with the orientation of the world frame.
```

Comment:
To clarify possible confusion with an API that permanently set the force (in the sense that after you set it, you need it to reset to zero) could it make sense to add:
```suggestion
        The torque is expressed with the orientation of the world frame.
        The external force and torque is only applied for the next simulation step,
        and then is automatically reset to zero. To apply a force for a more then one step, 
        call `apply_external_force` before each step.  
```

Reply:
Thanks! I agree that this is a important piece of information.

Reply:
Check this diff for the changes https://github.com/robotology/gym-ignition/compare/a98480d2d0ce9647f81ea7de49263d8fe2281611..af4c9d750f953f87da4eeede155e9ab9aa1ea43d

Reply:
> Check this diff for the changes https://github.com/robotology/gym-ignition/compare/a98480d2d0ce9647f81ea7de49263d8fe2281611..af4c9d750f953f87da4eeede155e9ab9aa1ea43d

Perfect!